apps/web/src/config/constants/tokenLists/pancake-supported-onramp-currency-list.json

```diff
@@ -49,6 +49,42 @@
       "decimals": 18,
       "chainId": 1,
       "logoURI": "https://tokens.pancakeswap.finance/images/symbol/dai.png"
+    },
+    {
+      "isNative": "false",
+      "isToken": "true",
```

Comment:
Why some tokens have `isNative` & `isToken` ?
But why `USD Coin Zksync` & `Tether USD Zksync` no this two?

Reply:
Prefer use the data from https://github.com/pancakeswap/token-list/pull/193

Reply:
i thought to add the chain prefix so that users are certain its eth on zk and not regular eth. cos name/symbols s are all same otherwise. i will revert this now if u prefer that.

Reply:
For me is better to use the URL from token-list.
So we only need to maintain that file.